services/tenant-ui/frontend/src/components/reservation/status/Approved.vue

```diff
@@ -88,7 +94,9 @@ const handleSubmit = async (isFormValid: boolean) => {
       formFields.password
     );
   } catch (error) {
-    toast.error(`Failure: ${error}`);
+    // TODO: Handle expired passwords as well
+    console.error(error);
+    showError.value = true;
```

Comment:
This will show invalid password on any type of error now (network anything, 5xx, Traction down). Think this should be specific to 401 or 403 type errors when the password is actually the issue.
Anyways when handling the expired password later might have to do a string check unless it returns a different error code.

I think it's should be able to just throw a specific error or include the response code in the check-in method

Reply:
I was trying to be generic. But we could enhance it with the expired option.
I'm not using docker. How can I adjust the expiry if I'm just linking to dev?

Reply:
Generic works for the user/pw/expiry cases but stuff like traction down or wrong network connection (or any client side error) might not want to trigger a "Your password is incorrect" so I'd throw that only when a 401 or 403 is detected.

Can handle expiry later, if you did want to try it out though but if you don't want to use local docker (would recommend getting that figured out though) then you could set the helm value on a PR to really low (and then undo the commit changing the PR value before merging) with `reservationExpiryMinutes` in the values-pr helm values file. Then config your local tenant UI to the PR.

Reply:
K.. I'm replicating the expired reservation errors now.
I'll show the inline message if there's a 401 and an expiry message... or if there's a 401 and an incorrect password note. Otherwise I'll send the error to Toast.

Reply:
That sounds good!